.github/data/distros.yml

```diff
@@ -287,7 +278,23 @@ include:
     packages:
       <<: *ubuntu_packages
       repo_distro: ubuntu/focal
+
 no_include: # Info for platforms not covered in CI
+  - distro: archlinux
+    version: latest
+    support_type: Intermediate
+    notes: ''
+    eol_check: false
+    env_prep: |
+      pacman --noconfirm -Syu && pacman --noconfirm -Sy grep libffi
+    test:
+      ebpf-core: true
```

Comment:
We do, in fact, cover Arch in our CI, so it needs to be up in the `include` section, not down here.